src/domUpdates.js

```diff
@@ -41,18 +49,37 @@ let domUpdates = {
     header.innerText = 'Available Rooms';
     let bidetMessage;
     availableRooms.forEach(room => {
-      room.bidet ? bidetMessage = 'Hooray there\'s a bidet!' : bidetMessage = 'No bidet for you'
+      room.bidet ? bidetMessage = 'Luxury Bidet' : bidetMessage = 'No bidet for you'
       availableRoomsSection.innerHTML +=
+      // `
+      // <article class="room-card" id="${room.number}">
+      //   <h4>${searchDate}</h4>
+      //   <h5>${room.roomType}</h5>
+      //   <p>Room Number ${room.number}</p>
+      //   <p>Beds: ${room.numBeds} ${room.bedSize}<p>
+      //   <p>$${room.costPerNight}/ night</p>
+      //   <p>Bidet included?</p>
+      //   <p>${bidetMessage}</p>
+      // </article>
+      // `
       `
-      <article class="room-card" id="${room.number}">
-        <h4>${searchDate}</h4>
-        <h5>${room.roomType}</h5>
-        <p>Room Number ${room.number}</p>
-        <p>Beds: ${room.numBeds} ${room.bedSize}<p>
-        <p>$${room.costPerNight}/ night</p>
-        <p>Bidet included?</p>
-        <p>${bidetMessage}</p>
-      </article>
+        <article class="room-card" id="${room.number}">
+          <div class="card-header">
+            <h4>Date Available:</h4>
+            <h4>${searchDate}</h4>
+          </div>
+          <div class="card-details">
+            <div class="booking-info">
+              <p>Room #${room.number}</p>
```

Comment:
very divvy. look into sections and dl, dt, dd semantics

Reply:
Agreed yes I'll try to take them out!